languages.toml

```
@@ -2117,3 +2117,23 @@ roots = []
 comment-token = "#"
 indent = { tab-width = 4, unit = "    " }
 grammar = "python"
+
+[[language]]
+name = "MSBuild"
+scope = "source.xml"
+injection-regex = "xml"
+file-types = [".proj", ".vbproj", ".csproj", ".fsproj", ".targets", ".props"]
+indent = { tab-width = 2, unit = "  " }
+roots = []
+
+[language.auto-pairs]
+'(' = ')'
+'{' = '}'
+'[' = ']'
+'"' = '"'
+"'" = "'"
+"<" = ">"
+
+[[grammar]]
+name = "MSBuild"
+source = { git = "https://github.com/RenjiSann/tree-sitter-xml", rev = "48a7c2b6fb9d515577e115e6788937e837815651" }
```

```suggestion
roots = []
grammar = "xml"

[language.auto-pairs]
'(' = ')'
'{' = '}'
'[' = ']'
'"' = '"'
"'" = "'"
"<" = ">"
```

This will re-use the xml tree-sitter grammar without adding another download.

---

thank you.  I have learned something.  Next commit has all the requested changes.